src/fonts.test.ts

```diff
@@ -0,0 +1,186 @@
+import { createFigma } from "figma-api-stub";
+import { Inter } from "figma-api-stub/dist/fonts";
+
+import * as F from "./figma-json";
+import {
+  dump,
+  fontsToLoad,
+  loadFonts,
+  fallbackFonts,
+  applyFontName,
+  encodeFont
+} from ".";
+
+const notInstalledFontFamily = "My Custom Font";
+
+beforeEach(() => {
+  (globalThis as any).figma = createFigma({});
+
+  // Replace figma-api-stub's mock with one that fails to load the not installed font family.
+  const loadFontAsyncMock = jest.fn(async (font: FontName) => {
+    return new Promise<void>((resolve, reject) => {
+      if (font.family === notInstalledFontFamily) {
+        reject("Font not found");
+      }
+
+      resolve();
```

Comment:
Looks like this should be `} else {` so as not to rely on the idea that resolve() after reject() is a no-op, right?

Reply:
Good call!